source/gui/settingsDialogs.py

```diff
@@ -3921,16 +3921,15 @@ def makeSettings(self, settingsSizer):
 		tables = brailleTables.listTables()
 		# Translators: The label for a setting in braille settings to select the output table (the braille table used to read braille text on the braille display).
 		outputsLabelText = _("&Output table:")
-		outTables = [table for table in tables if table.output]
-		self.outTableNames = [table.fileName for table in outTables]

```

Comment:
this is an API breaking change, we still need to maintain a list for outTableNames.
Perhaps consider deprecating.